homeassistant/components/media_player/__init__.py

```diff
@@ -536,16 +536,19 @@ def supported_features(self):
             # If this platform is still using supported_media_commands, issue
```

Comment:
Should there be a test case for this path with an old-style component during the transitional period?